server/src/main/java/com/tencent/rss/server/storage/MultiStorageManager.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available. 
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved. 
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.server.storage;
+
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.collect.Lists;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.tencent.rss.server.ShuffleDataFlushEvent;
+import com.tencent.rss.server.ShuffleDataReadEvent;
+import com.tencent.rss.server.ShuffleServerConf;
+import com.tencent.rss.server.ShuffleUploader;
+import com.tencent.rss.storage.common.LocalStorage;
+import com.tencent.rss.storage.common.Storage;
+
+public class MultiStorageManager implements StorageManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(MultiStorageManager.class);
+
+  private final StorageManager warmStorageManager;
+  private final StorageManager coldStorageManager;
+  private final List<ShuffleUploader> uploaders  = Lists.newArrayList();
+  private final boolean uploadShuffleEnable;
+  private final long flushColdStorageThresholdSize;
+
+  MultiStorageManager(ShuffleServerConf conf, String shuffleServerId) {
+    warmStorageManager = new LocalStorageManager(conf);
+    coldStorageManager = new HdfsStorageManager(conf);
+    uploadShuffleEnable = conf.get(ShuffleServerConf.UPLOADER_ENABLE);
+    flushColdStorageThresholdSize = conf.getSizeAsBytes(ShuffleServerConf.FLUSH_COLD_STORAGE_THRESHOLD_SIZE);
+    if (uploadShuffleEnable) {
+      if (!(warmStorageManager instanceof LocalStorageManager)) {
```

Comment:
Use a type may be more appropriate.

Reply:
What's your meaning?

Reply:
It's a minor suggestion, no need to fix.